src/tests/unittests/test_structures.py

```diff
@@ -34,7 +34,8 @@ def test_sensingdot_t(fig=None, verbose=0):
         sensing_dot = qtt.structures.sensingdot_t(
             ['SD1a', 'SD1b', 'SD1c'], station=station, minstrument='keithley1.amplitude')
         sensing_dot.verbose = verbose
-        _ = sensing_dot.autoTune(step=-8, fig=fig)
+        if fig is not None:
+            _ = sensing_dot.autoTune(step=-8, fig=fig)
```

Comment:
`autoTune` does more than plotting. So it should be executed in the unittest

Reply:
I copied the non-silent test to integration. Because all the windows popping up is crashing wildly on linux (python abort).
In the default pytest settings (pytest.ini) also the tests in directory integration are executed as unit test. On linux this pytest.ini can be adjusted to only test the 'silent' unit tests. When necessary we can add another pytest configuration file. One for integration tests and one for silent tests.